### PR TITLE
Updates devDependencies

### PR DESCRIPTION
**Pull request by BarryThePenguin**
Updates `devDependencies` for ember to `v3.24.0` using https://github.com/ember-cli/ember-cli-update

The new template includes some `eslint` changes, including the addition of `prettier`. So this PR also includes the result of running `yarn eslint --fix`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,9 +10,14 @@ const { parse, generatePreviewHead } = require('./util');
 module.exports = {
   name: require('./package').name,
 
+  configKey: 'ember-cli-storybook',
```
Comment (dbendaou) [inline index.js:13]: This can be removed from this PR :)

**Review by dbendaou — CHANGES_REQUESTED** [review]

**Comment by gabrielcsapo** [comment]
I will look to resolve these conflicts to have this merged.

File: index.js (hunk 1 above)
Comment (gabrielcsapo) [reply index.js:13]: Why do we want to remove this?

Comment (dbendaou) [reply index.js:13]: This was already handled in https://github.com/storybookjs/ember-cli-storybook/pull/54/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R13

**Comment by gabrielcsapo** [comment]
Dropping as we are on 3.27 now.

**Pull request closed without merging**